sklearn/ensemble/voting_classifier.py

```diff
@@ -133,9 +162,16 @@ def fit(self, X, y):
         self.classes_ = self.le_.classes_
         self.estimators_ = []
 
-        for name, clf in self.estimators:
-            fitted_clf = clone(clf).fit(X, self.le_.transform(y))
-            self.estimators_.append(fitted_clf)
+        transformed_y = self.le_.transform(y)
+
+        self.estimators_ = Parallel(n_jobs=self.n_jobs)(
+                delayed(_parallel_fit_estimator)(
+                    clone(clf),
```

Comment:
A few cosmetic issues here.
1. Let's avoid this level of nesting.
2. closing parentheses conventionally appear right after what they're closing, not on a new line.